geometry_msgs/PoseStamped goal
---
pips_trajectory_msgs/trajectory_points ni_traj
